components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -782,19 +782,19 @@ subroutine ocn_vel_vmix_tend_implicit_spatially_variable_mannings(meshPool, forc
           endif
           if (vegetationMask(iCell) .eq. 1) then
             total_h = bottomDepth(iCell) + ssh(iCell)
-            old_bottom_Cd = gravity * bottomDrag(iCell)**2.0_RKIND * total_h**(1.0_RKIND/3.0_RKIND)
+            old_bottom_Cd = gravity * bottomDrag(iCell)**2 * total_h**(-1.0_RKIND/3.0_RKIND)
```

Comment:
@xylar thanks for checking. The added negative sign is here, and is identical to the line in the MPAS PR: https://github.com/MPAS-Dev/MPAS-Model/pull/868/files Comparing the two, I think all the changes made it to the new PR.

Reply:
Oh, sorry.  I saw a bunch of 1/3 without the minus and obviously missed this one.